src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -1182,6 +1161,46 @@ components:
         message_hex:
           $ref: "#/components/schemas/HexString"
 
+    #
+    # /stream/* Request & Response Types
+    #
+
+    StreamTransactionsRequest:
+      allOf:
+        - $ref: "#/components/schemas/AtLedgerStateMixin"
+        - $ref: "#/components/schemas/CursorLimitMixin"
+        - type: object
+          properties:
+            transaction_type:
```

Comment:
probably I'm just missing some information but that wasn't part of endpoint specs. I'm asking because we're adding `epoch_change` as transaction type which isn't really a transaction type?

If i understand things right all transaction types can result in new epoch. Exposing such type to outer world might cause confusion that it's actual transaction type and it might be also confusing that user asked for transaction_type `epoch_change` and we're returning him all sort of different transaction types.

Reply:
I'll try to find out a different name to reduce confusion.